src/ctapipe/instrument/optics.py

```diff
@@ -263,3 +272,168 @@ def __repr__(self):
+    Used to calculate a pdf asymmetry parameter K of the asymmetric radial laplacian of the psf as
+    a function of the distance r to the optical axis
+
+    .. math:: K(r) = 1 - asym_0 \tanh(asym_1 r) - asym_2 r
+
+    radial_scale_params describes the dependency of the radial scale on the distance to the center of the camera
+    Used to calculate width Sr of the asymmetric radial laplacian in the PSF as a of function the distance r to the optical axis
+
+    .. math:: S_{R}(r) & = b_1 + b_2\,r + b_3\,r^2 + b_4\,r^3
+
+    az_scale_params Describes the dependency of the azimuthal scale on the distance to the center of the camera
+    Used to calculate the width Sf of the azimuthal laplacian in the PSF as a function of the angle :math:`phi`
+
+    .. math:: S_{\phi}(r) & = a_1\,\exp{(-a_2\,r)}+\frac{a_3}{a_3+r}
+
+    for reference see :cite:p:`startracker`
+    """
+
+    asymmetry_params = List(
+        help=(
+            "Describes the dependency of the psf on the distance"
+            "to the center of the camera. Used to calculate a pdf"
+            "asymmetry parameter K of the asymmetric radial laplacian"
+            "of the psf as a function of the distance r to the optical axis"
+        )
+    ).tag(config=True)
+
+    radial_scale_params = List(
+        help=(
+            "Describes the dependency of the radial scale on the"
+            "distance to the center of the camera Used to calculate"
+            "width Sr of the asymmetric radial laplacian in the PSF"
+            "as a of function the distance r to the optical axis"
+        )
+    ).tag(config=True)
+
+    az_scale_params = List(
+        help=(
+            "Describes the dependency of the azimuthal scale on the"
+            "distance to the center of the camera. Used to calculate"
+            "the the width Sf of the azimuthal laplacian in the PSF"
+            "as a function of the angle phi"
+        )
+    ).tag(config=True)
+
+    def __init__(
+        self,
+        subarray,
+        **kwargs,
+    ):
+        r"""
+         PSF model, describing purely the effect of coma aberration on the PSF
+        uses an asymmetric laplacian for the radial part
+
+        .. math:: f_{R}(r, K) = \begin{cases}\frac{1}{S_{R}(K+K^{-1})}e^{-K\frac{r-L}{S_{R}}}, r\ge L\\ \frac{1}{S_{R}(K+K^{-1})}e^{\frac{r-L}{KS_{R}}}, r < L\end{cases}
+
+        and a symmetric laplacian in azimuthal direction
+
+        .. math:: f_{\Phi}(\phi) = \frac{1}{2S_\phi}e^{-|\frac{\phi-\phi_0}{S_\phi}|}
+
+        Parameters
+        ----------
+        subarray: ctapipe.instrument.SubarrayDescription
+            Description of the subarray.
+        """
+        super().__init__(subarray=subarray, **kwargs)
+
+    def k_func(self, x):
+        return (
+            1
+            - self.asymmetry_params[0] * np.tanh(self.asymmetry_params[1] * x)
+            - self.asymmetry_params[2] * x
+        )
+
+    def sr_func(self, x):
+        return np.polyval(self.radial_scale_params, x)
```

Comment:
needs a `[::-1]` on the parameters as `polyval` goes from highest power to lowest power (Horner schema)